src/explore-education-statistics-common/src/components/FigureFootnotes.tsx

```diff
@@ -17,7 +17,12 @@ const FigureFootnotes = ({ footnotes, headingHiddenText }: Props) => {
           <VisuallyHidden>{` ${headingHiddenText}`}</VisuallyHidden>
         )}
       </h3>
-      <CollapsibleList listStyle="number" collapseAfter={2} testId="footnotes">
+      <CollapsibleList
+        listStyle="number"
+        collapseAfter={2}
+        id="footnotes"
```

Comment:
We can have many data blocks in release content, so this ID is almost certain to have collisions with other footnotes on the page.

Reply:
Good spot! Sorted.